lib/vsc/administration/user.py

```diff
@@ -151,12 +152,14 @@ def __init__(self, user_id, storage=None, pickle_storage=None, rest_client=None,
         if storage is None:
             storage = VscStorage()
 
+        # Initialze the corresponding operator for each storage backend
+        for fs in storage[self.host_institute]:
+            storage[self.host_institute][fs].operator = StorageOperator(storage[self.host_institute][fs])
```

Comment:
why not an instance of the operator? (it has been a while since i read this code ;)

Reply:
I'm not sure what you mean, this line is already initializing an instance of `StorageOperator`.